main.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+class rs_m
+{
+public:
+    vector<double> r; 
+
+    void setResort(vector<double> r)
+    {
+        this->r = r;
+    }
+
+    //getting the size of the resort
+    double GetResortSize() 
+    {
+        int len = r.size();
+        return len;
+    }
+
+    int show()
+    {
+        for (int i = 0; i < r.size(); i++)
+        {
+            cout << r[i] << " ";
+        }
+        cout << endl;
+    }
+};
+
+class RSMclient
+{
+public:
+    vector<string> c;
+
+    //setting clients
+    void setClient(vector<string> c)
+    {
+        this->c = c;
+    }
+
+    int GetClientCount(int k) 

```

Comment:
Again, the function name should follow the same convention as the whole code base - lower came case.
Also, is `k` parameter used anywhere?